src/main/java/io/appform/dropwizard/actors/base/UnmanagedPublisher.java

```diff
@@ -65,7 +65,17 @@ public final void publish(Message message) throws Exception {
     }
 
     public final void publish(Message message, AMQP.BasicProperties properties) throws Exception {
-        publishChannel.basicPublish(config.getExchange(), queueName, properties, mapper().writeValueAsBytes(message));
+        String routingKey;
+        if (config.isSharded()) {
```

Comment:
This is a repeated code at 2 places. Create a util method which can take actor config and name as input and determine final queue name